pkgs/dyne/zenroom/registry.yaml

```diff
@@ -4,21 +4,42 @@ packages:
     repo_owner: dyne
     repo_name: zenroom
     description: Embedded no-code VM executing human-like language to manipulate data and process cryptographic operations
-    version_constraint: "false"
+    version_constraint: semver("<= 3.10.0")
```

Comment:
You shouldn't change this version_constraint.
- https://github.com/aquaproj/aqua-registry/pull/36073#issuecomment-2850280577